test/e2e/node_feature_discovery_test.go

```diff
@@ -35,7 +35,6 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	clientset "k8s.io/client-go/kubernetes"
 	taintutils "k8s.io/kubernetes/pkg/util/taints"
-	"k8s.io/kubernetes/test/e2e/framework"
 	e2elog "k8s.io/kubernetes/test/e2e/framework"
```

Comment:
Drop the `e2elog` instead

```suggestion
	"k8s.io/kubernetes/test/e2e/framework"
```

and change all `e2elog.Logf` to `framework.Logf`

Reply:
@testwill , once this changes are in, I think we can move forward with this contribution.

thanks for your contribution to NFD!!